portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,40 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Dipti's Portfolio</h1>
+        <p>Hi! My name is Dipti and I'm a junior at Santa Clara University majoring in Computer Science and minoring in Digital Art. In general, I like to combine my two passions to create new things with good people. This can range from Virtual Reality games through research to web design with my friends. In my free time, I enjoy spending time with family and friends and watching YouTube and Netflix. </p>
+        <img src= "/images/IMG_0567.JPG" width = "200" height="250">
```

Comment:
You could scale these images proportionally by setting the width to your desired value and height to auto in CSS

https://www.w3schools.com/css/css_rwd_images.asp